ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/router_bgp.py

```diff
@@ -45,11 +45,12 @@ def _bgp_cluster_id(self) -> str | None:
                 return get(self._hostvars, "switch.bgp_cluster_id", default=self._router_id)
         return None
 
-    def _generate_base_peer_group(self, pg_type: str, pg_name: str) -> dict:
+    def _generate_base_peer_group(self, peer_group_name: str, pg_type: str, pg_name: str) -> dict:
```

Comment:
It seems like we already have the peer_group_name here, in pg_name. So no need to add it again. My mistake.